{"name": "test_mine[case5]", "status": "broken", "statusDetails": {"message": "Exception:   .元素不存在", "trace": "self = <test_mine.TestMine object at 0x000001B6A514AD48>\ncase = {'id': 'mine_006', '优先级': 'P1', '前置条件': '', '是否执行': 'yes', ...}\n\n    @pytest.mark.parametrize(\"case\", run_ed)\n    def test_mine(self, case):\n        # case_id = case[ExcelConfig.case_id]\n        # print(case_id)\n        # case_catalog = case[ExcelConfig.case_catalog]\n        # case_name = case[ExcelConfig.case_name]\n        # case_priority = case[ExcelConfig.case_priority]\n        # case_is_run = case[ExcelConfig.case_is_run]\n        # case_preconditions = case[ExcelConfig.case_preconditions]\n        # case_step = case[ExcelConfig.case_step]\n        case_step_node = case[ExcelConfig.case_step_node]\n        date1 = analysis(case_step_node)\n        # print(date1)\n        # case_expected_outcome = case[ExcelConfig.case_expected_outcome]\n        case_expected_node = case[ExcelConfig.case_expected_node]\n        data2 = expected_analysis(case_expected_node)\n        print(data2)\n        d = ExecuteU2(\"851QFDSG22CDZ\", \"com.tencent.hobby\")\n        for step_node, expected_node in zip(date1, data2):\n            d.run_app(step_node)\n            time.sleep(0.5)\n            # for expected_node in data2:\n            if type(expected_node) == str:\n>               d.app_assert(expected_node)\n\ntestcase\\test_mine.py:38: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <utils.uiauto_util.ExecuteU2 object at 0x000001B6A6646088>\nelement = '  '\n\n    def app_assert(self, element):\n        \"\"\"页面断言\"\"\"\n        time.sleep(0.7)\n        if self.dv(text=element).exists(timeout=5):\n            # exists: 等待元素出现\n            print(f\"{element}.success\")\n        elif self.dv(resourceId=element).exists(timeout=5):\n            print(f\"{element}.success\")\n        elif element == \" \":\n            pass\n        else:\n>           raise Exception(f\"{element}.元素不存在\")\nE           Exception:   .元素不存在\n\nutils\\uiauto_util.py:59: Exception"}, "parameters": [{"name": "case", "value": "{'id': 'mine_006', '用例目录': '影响力', '用例名称': '影响力为0时', '优先级': 'P1', '是否执行': 'yes', '前置条件': '', '步骤': '1.嚯app启动》点击底部【我的】tab，我的画面，内容没被点赞过，内容没被评论数，内容没被转发过，内容没被收藏过，评论没被点赞过，评论没被点赞过，点击影响力', '步骤节点': '我的, 2202005276啊啊, 影响力', '预期结果': '1.影响力的总数为0，并且内容被点赞数为0，内容被评论数为0，内容被转发数为0，内容被收藏数为0，评论被点赞数为0，评论被回复数为0', '预期节点': ' \\n  \\n我的影响力, 内容被点赞, 内容被评论, 内容被转发, 内容被收藏, 评论被点赞'}"}], "start": 1626765785625, "stop": 1626765854539, "uuid": "100e41dd-d316-4797-b94a-295b817b7c7a", "historyId": "38112308d51c6392a6ce8b9aace1f497", "testCaseId": "da5ae44c99fde797f954ed7865065f3e", "fullName": "testcase.test_mine.TestMine#test_mine", "labels": [{"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_mine"}, {"name": "subSuite", "value": "TestMine"}, {"name": "host", "value": "v_linkaica-PC0"}, {"name": "thread", "value": "7808-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_mine"}]}